### ISSUE TITLE
Skipped heading level

### ISSUE THREAD

**Issue by cdefy** [blueglassdigital/job-postings#55]
Hi,

Following @cfoellmann request on WP support forum, I post one of the plugin issue here. ([Topic on WP](https://wordpress.org/support/topic/skipped-heading-level/#new-topic-0))

It seems that the title in the modal is a h4 but as the other titles are h2, accessibility verification tools are not happy. I reckon it should be a h2 as well, coming at the same level as the h2 of the job-side titles.

Thanks in advance!
